lerobot/scripts/train.py

```diff
@@ -104,42 +101,39 @@ def update_policy(
     batch,
     optimizer,
     grad_clip_norm,
-    grad_scaler: GradScaler,
     lr_scheduler=None,
-    use_amp: bool = False,
+    accelerator=None,
 ):
     """Returns a dictionary of items for logging."""
     start_time = time.perf_counter()
-    device = get_device_from_parameters(policy)
+
     policy.train()
-    with torch.autocast(device_type=device.type) if use_amp else nullcontext():
-        output_dict = policy.forward(batch)
-        # TODO(rcadene): policy.unnormalize_outputs(out_dict)
-        loss = output_dict["loss"]
-    grad_scaler.scale(loss).backward()
-
-    # Unscale the graident of the optimzer's assigned params in-place **prior to gradient clipping**.
-    grad_scaler.unscale_(optimizer)
-
-    grad_norm = torch.nn.utils.clip_grad_norm_(
-        policy.parameters(),
-        grad_clip_norm,
-        error_if_nonfinite=False,
-    )
 
-    # Optimizer's gradients are already unscaled, so scaler.step does not unscale them,
-    # although it still skips optimizer.step() if the gradients contain infs or NaNs.
-    grad_scaler.step(optimizer)
-    # Updates the scale for next iteration.
-    grad_scaler.update()
+    output_dict = policy.forward(batch)
+    # TODO(rcadene): policy.unnormalize_outputs(out_dict)
+    loss = output_dict["loss"]
+    if accelerator:
+        accelerator.backward(loss)
```

Comment:
_Note_ : accelerate handles `mixed_precision` automatically on `accelerator.backward` : 
[huggingface.co/docs/accelerate/en/basic_tutorials/migration#mixed-precision](https://huggingface.co/docs/accelerate/en/basic_tutorials/migration#mixed-precision)

Reply:
I think I'm clearer on this now that you've moved to having all the accelerate options in the command line rather than in the yaml config. Now I understand your first comment. Unblocking.